pyproject.toml

```
@@ -39,6 +39,7 @@ dependencies = [
   "frozenlist >= 1.1.1",
   "multidict >=4.5, < 7.0",
   "propcache >= 0.2.0",
+  "typing_extensions >= 4.0 ; python_version < '3.13'",
```

```suggestion
  "typing_extensions >= 4.4 ; python_version < '3.13'",
```
https://github.com/python/typing_extensions/commit/fafb5c597f586e8109a1c668c706264060f5456a